package-lock.json

```diff
@@ -19766,9 +19769,9 @@
       "dev": true
     },
     "picomatch": {
-      "version": "2.2.2",
-      "resolved": "https://registry.npmjs.org/picomatch/-/picomatch-2.2.2.tgz",
-      "integrity": "sha512-q0M/9eZHzmr0AulXyPwNfZjtwZ/RBZlbN3K3CErVrk50T2ASYI7Bye0EvekFY3IP1Nt2DHu0re+V2ZHIpMkuWg=="
+      "version": "2.2.3",
+      "resolved": "https://registry.npmjs.org/picomatch/-/picomatch-2.2.3.tgz",
+      "integrity": "sha512-KpELjfwcCDUb9PeigTs2mBJzXUPzAuP2oPcA989He8Rte0+YUAjw1JVedDhuTKPkHjSYzMN3npC9luThGYEKdg=="
```

Comment:
This also includes the update of `picomatch`. Looks like problems. 🆗 

<details>
<summary><code>npm diff --diff=picomatch@2.2.2 --diff=picomatch@2.2.3</code></summary>

````diff
diff --git a/package.json b/package.json
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "picomatch",
   "description": "Blazing fast and accurate glob matcher written in JavaScript, with no dependencies and full support for standard and extended Bash glob features, including braces, extglobs, POSIX brackets, and regular expressions.",
-  "version": "2.2.2",
+  "version": "2.2.3",
   "homepage": "https://github.com/micromatch/picomatch",
   "author": "Jon Schlinkert (https://github.com/jonschlinkert)",
   "funding": "https://github.com/sponsors/jonschlinkert",
diff --git a/CHANGELOG.md b/CHANGELOG.md
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -32,6 +32,13 @@
 
 </details>
 
+## 2.2.3 (2021-04-10)
+
+### Fixed
+
+* Do not skip pattern seperator for square brackets ([fb08a30](https://github.com/micromatch/picomatch/commit/fb08a30)).
+* Set negatedExtGlob also if it does not span the whole pattern ([032e3f5](https://github.com/micromatch/picomatch/commit/032e3f5)).
+
 ## 2.2.2 (2020-03-21)
 
 ### Fixed
diff --git a/index.js b/index.js
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/index.js
+++ b/index.js
diff --git a/LICENSE b/LICENSE
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/LICENSE
+++ b/LICENSE
diff --git a/README.md b/README.md
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/README.md
+++ b/README.md
@@ -562,7 +562,7 @@
 
 Some characters that are used for matching in regular expressions are also regarded as valid file path characters on some platforms.
 
-To match any of the following characters as literals: `$^*+?()[]
+To match any of the following characters as literals: `$^*+?()[]`
 
 Examples:
 
diff --git a/lib/constants.js b/lib/constants.js
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/lib/constants.js
+++ b/lib/constants.js
diff --git a/lib/parse.js b/lib/parse.js
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/lib/parse.js
+++ b/lib/parse.js
@@ -247,7 +247,7 @@
         output = token.close = `)$))${extglobStar}`;
       }
 
-      if (token.prev.type === 'bos' && eos()) {
+      if (token.prev.type === 'bos') {
         state.negatedExtglob = true;
       }
     }
diff --git a/lib/picomatch.js b/lib/picomatch.js
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/lib/picomatch.js
+++ b/lib/picomatch.js
diff --git a/lib/scan.js b/lib/scan.js
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/lib/scan.js
+++ b/lib/scan.js
@@ -231,13 +231,15 @@
           isBracket = token.isBracket = true;
           isGlob = token.isGlob = true;
           finished = true;
-
-          if (scanToEnd === true) {
-            continue;
-          }
           break;
         }
       }
+
+      if (scanToEnd === true) {
+        continue;
+      }
+
+      break;
     }
 
     if (opts.nonegate !== true && code === CHAR_EXCLAMATION_MARK && index === start) {
diff --git a/lib/utils.js b/lib/utils.js
old mode 100777
new mode 100644
index v2.2.2..v2.2.3 
--- a/lib/utils.js
+++ b/lib/utils.js
````

</details>